packages/react/src/isomorphicClerk.ts

```diff
@@ -94,7 +95,7 @@ export default class IsomorphicClerk {
     this.frontendApi = frontendApi;
     this.publishableKey = publishableKey;
     this.proxyUrl = (options as ClerkConstructorOptions | undefined)?.proxyUrl;
-    this.domain = (options as ClerkConstructorOptions | undefined)?.domain;
+    this.domain = addClerkPrefix((options as ClerkConstructorOptions).domain);
```

Comment:
☁️ I think this change will cause to add `clerk.` twice. `this.domain` is passed as domain param to Clerk at `/packages/react/src/isomorphicClerk.ts#L133` which also prefixes the domain with `.clerk` from the change in this PR at `/packages/clerk-js/src/core/clerk.ts`.

Reply:
Yes you are correct, preparing a fix